swift/Tests/Blockchains/CardanoTests.swift

```diff
@@ -57,9 +57,85 @@ class CardanoTests: XCTestCase {
 
         let encoded = output.encoded
         XCTAssertEqual(encoded.hexString,
-            "83a40082825820554f2fd942a23d06835d26bbd78f0106fa94c8a551114a0bef81927f66467af000825820f074134aabbfb13b8aec7cf5465b1e5a862bde5cb88532cc7e64619179b3e76701018282583901558dd902616f5cd01edcc62870cb4748c45403f1228218bee5b628b526f0ca9e7a2c04d548fbd6ce86f358be139fe680652536437d1d6fd51a006acfc082583901df58ee97ce7a46cd8bdeec4e5f3a03297eb197825ed5681191110804df22424b6880b39e4bac8c58de9fe6d23d79aaf44756389d827aa09b1a000ca99d021a000298a3031a032dcd55a100818258206d8a0b425bd2ec9692af39b1c0cf0e51caa07a603550e22f54091e872c7df29058403c1cbde706d10550f5f966a5071e9e01e18f7b555626ed7616a214bd951d29d0ff0458edf5c555eefe4d4280302160149fc6e7e56a4298a368b7ab357643190df6")
+            "83a40082825820554f2fd942a23d06835d26bbd78f0106fa94c8a551114a0bef81927f66467af000825820f074134aabbfb13b8aec7cf5465b1e5a862bde5cb88532cc7e64619179b3e76701018282583901558dd902616f5cd01edcc62870cb4748c45403f1228218bee5b628b526f0ca9e7a2c04d548fbd6ce86f358be139fe680652536437d1d6fd51a006acfc082583901df58ee97ce7a46cd8bdeec4e5f3a03297eb197825ed5681191110804df22424b6880b39e4bac8c58de9fe6d23d79aaf44756389d827aa09b1a000ca96c021a000298d4031a032dcd55a100818258206d8a0b425bd2ec9692af39b1c0cf0e51caa07a603550e22f54091e872c7df29058407cf591599852b5f5e007fdc241062405c47e519266c0d884b0767c1d4f5eacce00db035998e53ed10ca4ba5ce4aac8693798089717ce6cf4415f345cc764200ef6")
 
         let txid = output.txID
-        XCTAssertEqual(txid.hexString, "efcf8ec01fb8cd32bc5289c609c470b473cc79bc60b0667e1d68dc3962df1082")
+        XCTAssertEqual(txid.hexString, "9b5b15e133cd73ccaa85307d2986aebc846505118a2eb4e6111e6b4b67d1f389")
+    }
+
+    public static func minAdaAmount(tokenBundle: Data) -> UInt64 {
+        let tokenBundleData = TWDataCreateWithNSData(tokenBundle)
+        defer {
+            TWDataDelete(tokenBundleData)
+        }
+        return TWCardanoMinAdaAmount(tokenBundleData)
```

Comment:
Only for test? or app needs this too?

Reply:
This is a helper for calling TWCardanoMinAdaAmount, app would need it too, so maybe it should be put in some helper swift.